src/api-explorer/index.markdown

```diff
@@ -3,11 +3,18 @@ title: Swagger API Explorer
 layout: reference
 ---
 
-# Swagger API Explorer
+Browse [Swagger](https://swagger.io/) documents, such as what APIs are available, and their resource and method details.
 
-Browse swagger documents for v3.0 and 3.1 APIs, such as what APIs are available, the resource and method details for each API.
+## Base URIs
 
-## Base URLs
-* **US:** https://www.concursolutions.com/api/{version}
-* **EMEA:** https://eu1.concursolutions.com/api/{version}
+For APIs using the new OAuth2 framework:
 
+* **US**: https://us.api.concursolutions.com/api/{version}
```

Comment:
These URLs should not end with /api/{version} as the new v4 URLs don't use that format.